renderer/components/utils.js

```diff
@@ -5,3 +5,13 @@ export const openInBrowser = (url, event) => {
   event.preventDefault()
   shell.openExternal(url)
 }
+
+export const parseTestKeys = (testKeys) => {
+  let parsedTestKeys = null
+  try {
+    parsedTestKeys = JSON.parse(testKeys)
+  } catch (e) {
+    parsedTestKeys = null
```

Comment:
Do we maybe want to do something with this error? Perhaps logging it to stdout?

Also is it perhaps not redundant to set this again as null, since you have already initialised it as null?

Reply:
I was not familiar with `electron-log`. Didn't want to use `console.log`, so reassigned `null`.